src/test/java/uk/gov/hmcts/reform/sendletter/logging/AppInsightsTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.sendletter.logging;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.TelemetryConfiguration;
+import com.microsoft.applicationinsights.telemetry.TelemetryContext;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AppInsightsTest {
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Mock
+    private TelemetryClient telemetry;
+
+    private final TelemetryContext context = new TelemetryContext();
+
+    @Before
+    public void setUp() {
+        when(telemetry.getContext()).thenReturn(context);
+    }
+
+    @After
+    public void tearDown() {
+        reset(telemetry);
+    }
+
+    @Test
+    public void should_fail_creating_app_insights_due_to_requirement_miss() {
```

Comment:
perhaps 'when insrumentation key is not present' instead of vague 'requirement miss'?